contrib/syslog-ng-debun

```diff
@@ -1243,6 +1264,7 @@ setup_env_solaris () {
 	netstatlunp () { netstat -P udp -na ; }
 	myplimit () { plimit $1 ; }
 	free () { prtconf | grep Mem ; printf Pagesize:\  ; pagesize -a ; }
+	timestamp () { /usr/bin/perl -e 'print time, "\n";' ; }
```

Comment:
Why the `timestamp` re-defined here ?

Reply:
The script has to run on multiple different platforms. This means that we have to handle somehow when a given OS behaves differently than the others. That is why we introduced OS-specific re-definitions of shell functions or variables.

In this specific case, Solaris does not support the date +%s way of getting the unix timestamp, so we had to override the default way of doing it.

Reply:
Ok, one minor thing the path of `perl` I think should not be absolute (at least not hard coded absolute path), see also the `sed_equivalent_cmd`, where `perl` is relative.

Reply:
Corrected.